internal/connector/connector.go

```diff
@@ -63,22 +61,6 @@ type ListenerOptions struct {
 	Metrics string
 }
 
-type KubernetesOptions struct {
-	// AuthToken may be used to override the token used to authenticate with the
-	// kubernetes API server. When the connector is run in-cluster, the
-	// service account associated with the pod will be used by default.
-	// When run outside of cluster there is no default, and this value must
-	// be set to a token that has permission to impersonate users in the cluster.
-	AuthToken types.StringOrFile
```

Comment:
Does this change mean its not possible to run the connector outside of Kubernetes anymore?

Reply:
It should be, it's just that we have to use the kubectl config instead of our own config file.

Reply:
It's easier now: running `infra connector` will just read the current kubeconfig for auth info